CMakeLists.txt

```diff
@@ -18,11 +18,10 @@ endif()
 # s2geometry needs to use the same C++ standard that absl used to avoid
 # undefined symbol errors since ABSL_HAVE_STD_STRING_VIEW etc will
 # end up defined differently.  There is probably a better way to achieve
-# this than assuming what absl used. 
-# Using CACHE allows the user to override the default.
-set(CMAKE_CXX_STANDARD 11 CACHE STRING "The C++ standard to build with")
+# this than assuming what absl used.
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Now it cannot be set from outside!
Please revert this

Reply:
I probably now why it was written like this.

Change default value (cached) not change it for already existed cmakecache (if I correct) so maybe author faced with such issue, and remove cache from this line (instead of remove "local" cmakecache)

Reply:
Nope. Fixed.